combat-trainer.lic

```diff
@@ -3360,16 +3372,24 @@ class GameState
     Flags.reset('ct-damage-ready')
   end
 
+  def set_barb_balance_flag
```

Comment:
Follow-up question: are any of these `set*flag` functions needed due to this line? https://github.com/rpherbig/dr-scripts/blob/master/combat-trainer.lic#L3439

Reply:
I added def set_barb_balance_flag because there were defs for the other flags, I dont actually see where its called though.

  def set_barb_damage_flag
    Flags.reset('ct-damage-ready')
  end

  def set_barb_balance_flag
    Flags.reset('ct-balance-ready')
  end